packages/rocketchat-setup-wizard/server/lib/getWizardSettings.js

```diff
@@ -0,0 +1,15 @@
+Meteor.methods({
+	getWizardSettings(user) {
+		if (!user && !Meteor.userId()) {
+			return false;
+		}
+
+		user = Meteor.userId();
+
+		if (RocketChat.authz.hasRole(user, 'admin') && RocketChat.models && RocketChat.models.Settings) {
```

Comment:
If you are using a method to get the settings and only admins can access that settings why the new settings are public?

Reply:
Because the settings should appear on the Admin, so the admin user can change later if needed.